systemtest/src/test/java/io/strimzi/systemtest/ConnectClusterIT.java

```diff
@@ -145,4 +148,28 @@ public void testKafkaConnectScaleUpScaleDown() {
             assertThat(events, hasNoneOfReasons(Failed, Unhealthy, FailedSync, FailedValidation));
         }
     }
+
+    @Test
+    @JUnitGroup(name = "regression")
+    @ConnectCluster(name = CONNECT_CLUSTER_NAME, connectConfig = CONNECT_CONFIG)
+    public void testForUpdateValuesInConnectCM() {
```

Comment:
I think we should also test updates to `connect-config` changes here. Or do you plan separate test case for it?